static/js/common/upload-addon.js

```diff
@@ -395,6 +394,14 @@
                         });
                     }, 1000);
                 } else {
+                    if (results.addon_type==10) {
```

Comment:
Ugh, hardcoding a python constant value in a js...

Reply:
I could change it to be a string?  Tbh, this was a workaround until the validator knows about static themes - it generally detects and sends along the type in the validation response.  (The validator sends both the numeric constant and a string, iirc)

Reply:
I'd prefer a string like the ones we have in the API (http://addons-server.readthedocs.io/en/latest/topics/api/addons.html#addon-detail-type) but this can wait until this is more fleshed out.